requirements/base.txt

```diff
@@ -13,4 +13,6 @@ ordered-set==2.0.1                  # MIT
 Markdown==2.6.6    					# BSD
 
 -e git+https://github.com/edx/opaque-keys.git@d45d0bd8d64c69531be69178b9505b5d38806ce0#egg=opaque-keys
+# custom opaque-key implementations for ccx
+git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
```

Comment:
I like this idea and tried it out, but unfortunately ran into a dependency error with the `six` python package.  I believe it should be a requirement of opaque keys because it's being used, https://github.com/edx/opaque-keys/blob/master/opaque_keys/__init__.py#L13.  However, it's not in the requirements,  https://github.com/edx/opaque-keys/blob/master/test-requirements.txt.

I'm asking around about it right now.  What do you think about a follow up upgrade once we get that sorted out?